src/utils/getResponse.ts

```diff
@@ -31,10 +31,6 @@ export const getResponse = async <
   handlers: H,
 ): Promise<ResponsePayload> => {
   const relevantHandlers = handlers
-    .filter((requestHandler) => {
```

Comment:
I think we can keep this here as a fast return?

Reply:
Sounds, good, will do.